docs/v2-system-addon/vendor.md

```diff
@@ -0,0 +1,44 @@
+# Vendoring in npm libraries
+
+#### Build system basics
+
+#### Vendored libraries as externals
+
+#### Is this library a good candidate to depend on and vendor?
+
+#### How to vendor in a library
+
+- [ ] file a bug for vendoring in the library itself
```

Comment:
Can this also be at minimum its own commit in a PR.

Reply:
I think this wants to be its own bug, mostly for documentation reasons.  If someone wants to figure out why something landed, they should be able to easily get details from searching Bugzilla.  That said, I suppose that bug could just point to an existing PR, as long as the thing is in its own commit.

Reply:
Do you have an example bug? This is different than the license bug right?

Reply:
Filed one https://bugzilla.mozilla.org/show_bug.cgi?id=1570766, @dmose is that good?

Reply:
Looks good to me.